packages/core/src/components/Accordion/Item.tsx

```diff
@@ -13,6 +13,8 @@ export type Props = {
   id: string;
   /** Index amongst a collection of accordion items. */
   index?: number;
+  /** Removes horizontal padding from item title and both horizontal and vertical padding from item body. */
```

Comment:
to go with my other comment, update this to:

```
/** Removes horizontal the item and top padding from item body. */
```

Reply:
Done !